src/app/lib/utilities/preprocessor/rules/groups.js

```diff
@@ -0,0 +1,16 @@
+import deepGet from '../../../../helpers/json/deepGet';
+
+/* eslint-disable import/prefer-default-export */
+export const filterEmptyGroupItems = jsonRaw => {
```

Comment:
This can be a single rule, named with it being a default export :)

Reply:
I was thinking more like. We might have other `rules` for groups

Reply:
Yeah, I think each one in its own file is fine, nice and easy to see each rule in isolation IMO